dir.targets

```diff
@@ -28,5 +28,15 @@
     <TargetFrameworkProfile></TargetFrameworkProfile>
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
+
```

Comment:
I think the TargetOS part is already included, but @Priya91 is also working merging another change with https://github.com/dotnet/corefx/pull/7255.